### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Swan**
Looks a bit better and adds more information about the game + links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,63 @@
+# Quaver 
+
+[![Build Status](https://travis-ci.com/Quaver/Quaver.svg?branch=develop)](https://travis-ci.com/Quaver/Quaver) [![CodeFactor](https://www.codefactor.io/repository/github/swan/quaver/badge)](https://www.codefactor.io/repository/github/swan/quaver) [![license](https://img.shields.io/badge/license-Mozilla%20Public%20License%202.0-blue)](https://github.com/Quaver/Quaver/blob/develop/LICENSE) [![Discord](https://discordapp.com/api/guilds/354206121386573824/widget.png?style=shield)](https://discord.gg/nJa8VFr) <a href="https://twitter.com/intent/follow?screen_name=QuaverGame">
+        <img src="https://img.shields.io/twitter/follow/QuaverGame?style=social&logo=twitter"
+            alt="follow on Twitter"></a> [![stargazers](https://img.shields.io/github/stars/Quaver/Quaver?style=social)](#)
```
Comment (YaLTeR) [inline README.md]: Is the star count really necessary? It's displayed at the top right of the page.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,63 @@
...
+
 <p align="center"> 
-  <img src="https://i.imgur.com/UK3PWJW.png" width="175px" height="175px">
+  <img src="https://i.imgur.com/AohWq5l.png">
 </p>
 
-# Quaver [![Build Status](https://travis-ci.com/Quaver/Quaver.svg?branch=develop)](https://travis-ci.com/Quaver/Quaver) [![CodeFactor](https://www.codefactor.io/repository/github/swan/quaver/badge)](https://www.codefactor.io/repository/github/swan/quaver) [![Blog](https://img.shields.io/badge/Blog-Read-blue.svg)](https://blog.quavergame.com) [![Discord](https://discordapp.com/api/guilds/354206121386573824/widget.png?style=shield)](https://discord.gg/nJa8VFr)
+| [![steam](https://i.imgur.com/rR4p9mW.png)](https://store.steampowered.com/app/980610/Quaver/) | [![twitter](https://i.imgur.com/Kd0SXlh.png)](https://twitter.com/QuaverGame) | [![discord](https://i.imgur.com/YrBmbEf.png)](https://discord.gg/nJa8VFr) | [![blog](https://i.imgur.com/o0fq6MA.png)](https://blog.quavergame.com) | [![website](https://i.imgur.com/svZCnuI.png)](https://quavergame.com) | 
+| --- | --- | --- | --- | --- |
+| **Steam Store Page** | **Follow Us On Twitter** | **Join The Discord** | **Read The Developer Blog** | **Visit The Website** |
```
Comment (YaLTeR) [inline README.md]: image
Could you change the text so it all fits on one line?

File: README.md [hunk 3]
```diff
@@ -1,26 +1,63 @@
+# Quaver 
+
+[![Build Status](https://travis-ci.com/Quaver/Quaver.svg?branch=develop)](https://travis-ci.com/Quaver/Quaver) [![CodeFactor](https://www.codefactor.io/repository/github/swan/quaver/badge)](https://www.codefactor.io/repository/github/swan/quaver) [![license](https://img.shields.io/badge/license-Mozilla%20Public%20License%202.0-blue)](https://github.com/Quaver/Quaver/blob/develop/LICENSE) [![Discord](https://discordapp.com/api/guilds/354206121386573824/widget.png?style=shield)](https://discord.gg/nJa8VFr) <a href="https://twitter.com/intent/follow?screen_name=QuaverGame">
+        <img src="https://img.shields.io/twitter/follow/QuaverGame?style=social&logo=twitter"
```
Comment (YaLTeR) [inline README.md]: Perhaps remove Twitter and Discord from here now that you added links below?

File: README.md [hunk 4]
```diff
@@ -1,26 +1,65 @@
...
+| [![steam](https://i.imgur.com/rR4p9mW.png)](https://store.steampowered.com/app/980610/Quaver/) | [![twitter](https://i.imgur.com/Kd0SXlh.png)](https://twitter.com/QuaverGame) | [![discord](https://i.imgur.com/YrBmbEf.png)](https://discord.gg/nJa8VFr) | [![blog](https://i.imgur.com/o0fq6MA.png)](https://blog.quavergame.com) | [![website](https://i.imgur.com/svZCnuI.png)](https://quavergame.com) | 
+| --- | --- | --- | --- | --- |
+| **Steam Store Page** | **Follow Us On Twitter** | **Join The Discord** | **Read The Developer Blog** | **Visit The Website** |
+
+## About 
+
+**Quaver** is a community-driven and open-source competitve vertical scrolling rhythm game with two game modes and online leaderboards. It also features an in-game editor and a flexible multiplayer mode.
+
+It is also being officially released on [Steam](https://store.steampowered.com/app/980610/Quaver/) for Windows, Mac, and Linux - making it one of the most accessible community-driven rhythm games to date.
```
Comment (YaLTeR) [inline README.md]:
```suggestion
It is also being officially released on [Steam](https://store.steampowered.com/app/980610/Quaver/) for Windows, Mac, and Linux—making it one of the most accessible community-driven rhythm games to date.
```

File: README.md [hunk 5]
```diff
@@ -1,26 +1,65 @@
...
+
+## Status
+
+This project is currently under heavy development and is in an alpha state, however it is extremely playable as a standalone game. Quaver still has a ton of development ahead of it and will continue to get better as time goes on with new features and bug fixes being added in each day.
```
Comment (YaLTeR) [inline README.md]: "a ton of development": the word "development" was already used in the previous sentence, replace. "work" perhaps?

"being added in each day": "being added every day", also lol do we really add features and bug fixes every day

File: README.md [hunk 6]
```diff
@@ -1,26 +1,65 @@
...
+
+Occasionally we open registrations where you can join the Steam alpha and get exclusive access to all the features Quaver has to offer including online rankings, multiplayer, in-game chat, map downloading/uploading, and so much more. Registrations are **currently closed,** however you can still enjoy a limited version of Quaver by compiling the `develop` branch or by downloading one of the [GitHub Releases](https://github.com/Quaver/Quaver/releases).
```
Comment (YaLTeR) [inline README.md]: Are we planning more alpha regs tho? If not maybe remove or reword this.

File: README.md [hunk 7]
```diff
@@ -1,26 +1,65 @@
...
+
+## Features
+
+Although there are many games like this, **Quaver** offers many unique features and expands on the ideas of its predecessors that make it much different. It is also heavily inspired by a variety of our favorite rhythm games.
+
+* **Two Game Modes** - Play the game with 4 keys or challenge yourself with 7. Each game mode has separate global and country leaderboards to compete on.
+
+* **Global & Country Leaderboards** - Compete with your country and the world for the highest scores and aim to be #1.
```
Comment (YaLTeR) [inline README.md]: I see what you mean by the word "scores" but it sounds funny considering that score is irrelevant for this

File: README.md [hunk 8]
```diff
@@ -1,26 +1,65 @@
...
+
+* **Ranked & Casual Multiplayer** - Challenge up to 16 players in casual or competitive-oriented multiplayer battles. Multiplayer includes three game modes including Free-For-All, Teams, and Battle Royale.
```
Comment (YaLTeR) [inline README.md]: includes including

File: README.md [hunk 9]
```diff
@@ -1,26 +1,65 @@
...
+
+* **Map Editor** - Create your own maps to any of your favorite songs. Upload and share them with the world, and submit them for official ranking.
+
+* **Advanced Skinning System** - Completely customize your gameplay experience with the ability to create skins. Export and share your skins with friends or upload them to the Steam workshop.
```
Comment (YaLTeR) [inline README.md]:
```suggestion
* **Advanced Skinning System** - Completely customize your gameplay experience with skins. Export and share your skins with friends or upload them to the Steam workshop.
```

File: README.md [hunk 10]
```diff
@@ -1,26 +1,65 @@
...
+
+* **Replays** - Go back in time by watching your previous scores. Watch replays from other players around the world, or export your own and share them with your rivals.
+
+* **Over 10+ Game Modifiers** - Switch up the way you play by activating in-game modifiers. Customize the speed of the song, get rid of all the long notes, or even randomize the entire map!
```
Comment (YaLTeR) [inline README.md]: Well, I wouldn't go as far as calling the lane randomization "randomize the _entire map_"

File: README.md [hunk 11]
```diff
@@ -1,26 +1,65 @@
...
+
+* **Steam Achievements** - Become a master of **Quaver** by completing challenges and earning achievements!
```
Comment (YaLTeR) [inline README.md]: Why is Quaver bold here?

File: README.md [hunk 12]
```diff
@@ -1,26 +1,65 @@
...
+
+* **Play Maps From Other Games** - Coming from a different game and miss all of your favorite maps? Quaver supports both .osz and .sm files out of the box - with support for more games to be added in the future!
```
Comment (YaLTeR) [inline README.md]:
```suggestion
* **Play Maps From Other Games** - Coming from a different game and miss all of your favorite maps? Quaver supports both .osz and .sm files out of the box—with support for more games to be added in the future!
```

File: README.md [hunk 13]
```diff
@@ -1,26 +1,65 @@
...
+
+* **Join a Growing Community** - This game is 100% community-driven and built from the ground up with players' feedback in mind.
 
->🎶 The ultimate, community-driven, and open-source competitive rhythm game.
+* **Anyone Can Contribute** - Submit feedback, discuss with the developers, and see your ideas come to life in-game.
```
Comment (YaLTeR) [inline README.md]: Do these last two belong in "Features"?

File: README.md [hunk 14]
```diff
@@ -1,26 +1,65 @@
...
-* [Requirements](https://github.com/Quaver/Quaver#requirements)
-* [Compiling](https://github.com/Quaver/Quaver#compiling)
-* [Contributing](https://github.com/Quaver/Quaver#contributing)
+## Building & Running
 
-# Compiling
-Getting started with Quaver development is super easy. 
+Getting started with **Quaver** development is extremely easy.
 
 * Install the [.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)
```
Comment (YaLTeR) [inline README.md]: Isn't it 2.2?

File: README.md (hunk 2 above)
Comment (YaLTeR) [reply README.md]: Also pls use higher res images, or even better svgs, they look blurry on my 2x scale screen

File: README.md (hunk 10 above)
Comment (Swan) [reply README.md]: Then what? It's a copy-paste from the Steam page, so it should be updated in both places

File: README.md (hunk 14 above)
Comment (Swan) [reply README.md]: Still uses 2.1

**Pull request merged**